lucille-plugins/lucille-api/src/main/java/com/kmwllc/lucille/endpoints/StandardErrorResponse.java

```diff
@@ -0,0 +1,33 @@
+package com.kmwllc.lucille.endpoints;
+
+import java.util.HashMap;
+import java.util.Map;
+import jakarta.ws.rs.core.Response;
+
+public class StandardErrorResponse {
```

Comment:
I don't see any places where we create a new StandardErrorResponse, so there also aren't any places where we call getStatus() or getMessage()...
Also, this class doesn't implement Response so the name StandardErrorResponse is somewhat confusing

I see that the static buildReponse method is used in many places

Please rename this class to ResponseUtils
Please remove the instance variables and accessors and make the constructor private with no args
Can rename buildResponse to buildErrorResponse so a call would look like ResponseUtils.buildErrorResponse()

Reply:
:+1: done